.github/CODEOWNERS

```diff
@@ -508,6 +508,8 @@ tests/sentry/api/endpoints/test_organization_dashboard_widget_details.py     @ge
 **/issues/**                                                @getsentry/issues
 *code_mappings*.py                                          @getsentry/issues
 /src/sentry/api/helpers/actionable_items_helper.py          @getsentry/issues
+/src/sentry/search/events/builder/issue_platform.py         @getsentry/issues
+/src/sentry/search/events/builder/errors.py                 @getsentry/issues
```

Comment:
nit: can these get moved down before `/src/sentry/search/snuba/` to keep this section alphabetized?